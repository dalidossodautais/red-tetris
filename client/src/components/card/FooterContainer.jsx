import styled from "styled-components";

export default styled.div`
  align-self: flex-end;
  display: flex;
  height: 60px;
  justify-content: space-between;
  margin-top: 5px;
`;
