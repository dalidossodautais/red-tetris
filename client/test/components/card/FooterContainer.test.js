import React from "react";
import renderer from "react-test-renderer";
import FooterContainer from "../../../src/components/card/FooterContainer";

describe("FooterContainer", () => {
  test("Standard", () => {
    const component = renderer.create(<FooterContainer />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
