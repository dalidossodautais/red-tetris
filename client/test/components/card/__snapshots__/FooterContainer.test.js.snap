// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContainer Standard 1`] = `
<div
  className="sc-bdfBwQ kXeqYA"
/>
`;

exports[`FooterContainer Standard 2`] = `
<div
  className="sc-bdfBwQ kXeqYA"
/>
`;
